app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def framework_template_url_for(task_id:)
+    framework = API::Task.includes(:framework).find(task_id).first.try(:framework)
+
+    return if framework.nil?
+
+    "/templates/#{framework.short_name} MISO Data Template (August 2018).xls"
```

Comment:
This is a nice way of getting this out without too much work! I've made a note to go back and change this before the end of August!

However - the filename relates to the period the return is for - so in this case it's `July 2018` rather than `August 2018` since users are submitting for the month just finished.